templates/archives/stories/whirlwind.jinja2

```diff
@@ -0,0 +1,66 @@
+{% extends 'archives/story.jinja2' %}
+
+{% block story_title %}
+The Rise and Fall of Project Whirlwind
+{% endblock %}
+
+{% block story_credit %}
+Meesue Kim
+{% endblock %}
+
+
+{% block story_body %}
+<p>
+    Arguably the first large-scale and high-speed computer, the Whirlwind was developed by the MIT Servomechanisms
+    Laboratory in a project directed by Jay Forrester for the United States Navy. When the project started in 1944, it
+    was initially meant to design an aircraft simulator during World War II, but researchers soon realized that the
+    system was too slow and inaccurate. As a result of this, the simulation did not give a realistic enough feel for it
+    to be used as a proper training method for pilots. A key part of this project was the replacement of electrostatic
+    storage tubes with the
+    <a href="https://infinitehistory.mit.edu/video/edward-murrows-see-it-now%E2%80%94jay-forrester-and-whirlwind-computer-1951">
+        magnetic core memory,
+    </a>
+    which allowed for high speeds and reliability, just not enough for the computer to function properly as a
+    universal flight simulator. Because of the use of the magnetic core memory, Whirlwind became a fully functioning
+    computer that had the potential to be
+    <a href="http://127.0.0.1:8000/archives/doc/1_2_project_proposal_contract_2">
```

Comment:
Should just use path in the URL links, as local servers won't work once the site is live.